typescript/core/src/common/hasMcapPrefix.ts

```diff
@@ -0,0 +1,9 @@
+import { MCAP0_MAGIC } from "../v0/constants";
```

Comment:
can we also
- rename `MCAP0_MAGIC` to `MCAP_MAGIC`
- rename `DETECT_VERSION_BYTES_REQUIRED` to something else like `HAS_MCAP_PREFIX_BYTES_REQUIRED` (or maybe delete it, and just use `MCAP_MAGIC.length`?)
- remove the `v0` folder, etc.